pkg/devfile/adapters/kubernetes/storage/utils.go

```diff
@@ -112,3 +111,24 @@ func GetExistingPVC(Client *kclient.Client, volumeName, componentName string) (s
 		return "", err
 	}
 }
+
+// DeleteOldPVCs deletes all the old PVCs which are not in the processedVolumes map
+func DeleteOldPVCs(Client *kclient.Client, componentName string, processedVolumes map[string]bool) error {
+	label := "component=" + componentName
```

Comment:
this is kinda weird. Better if we use fmt.Sprinft? Ex. label := `fmt.Sprintf("component=%s", componentName)`